test/e2e/framework.go

```diff
@@ -568,7 +568,6 @@ func (data *TestData) deployAntreaIPSec() error {
 func (data *TestData) deployAntreaFlowExporter(ipfixCollector string) error {
 	// Enable flow exporter feature and add related config params to antrea agent configmap.
 	ac := []configChange{
-		{"FlowExporter", "true", true},
```

Comment:
we should also skip the e2e flow aggregator tests if the Flow Exporter feature is disabled.